app/models/shelter.rb

```diff
@@ -16,6 +16,29 @@ def self.order_by_number_of_pets
       .order("pets_count DESC")
   end
 
+  def self.reverse_alpha 
+    find_by_sql(
+      "SELECT shelters.*
+      FROM shelters 
+      ORDER BY shelters.name DESC"
+    )
+  end
+
+  def self.pending_apps 
+    select("shelters.*")
+      .distinct
+      .joins(pets: { pet_applications: :application })
```

Comment:
Nice work! These queries look great. Another option, since our connection between pets and applications is set up through associations, is to join them directly like so`.joins(pets: :applications` (i.e. you don't need to mention pet_applications since you have a `has_many x through pet_applications` in the model already)